app/src/main/res/layout-sw600dp-port/onboarding_slide_final.xml

```diff
@@ -20,7 +20,7 @@
       android:layout_height="0dp"
       android:contentDescription="@string/onboarding_slide_3_title"
       android:src="@drawable/ic_portrait_onboarding_3"
-      app:layout_constraintDimensionRatio="5:4"
+      app:layout_constraintDimensionRatio="H, 384:307"
```

Comment:
This ratio should be `384:295`

Reply:
Done

Reply:
Then in that case the ratio should be 10:9 because image follows that ratio.